.github/workflows/ci.yml

```diff
@@ -164,6 +209,63 @@ jobs:
           cmake --build . --target clang clang-repl --parallel $(nproc --all)
         fi
         cd ../../
+
+    - name: Build LLVM/Cling on macOS if the cache is invalid
```

Comment:
This step is not really different from the step on unix. Can we merge them?

Reply:
Yaa so I tried combining them and faced an unforeseen error where building of llvm got terminated unexpectedly. Hence looking for it.